main.go

```diff
@@ -1,54 +1,48 @@
 package main
 
 import (
-	"errors"
 	"fmt"
-	"io"
-	"net/http"
-	"os"
-	"path/filepath"
-	"time"
-
+	"github.com/bitrise-io/go-steputils/stepconf"
 	"github.com/bitrise-io/go-utils/command"
 	"github.com/bitrise-io/go-utils/log"
 	"github.com/bitrise-io/go-utils/pathutil"
 	"github.com/bitrise-io/go-utils/retry"
+	"github.com/bitrise-io/go-xamarin/constants"
 	"github.com/bitrise-tools/go-steputils/cache"
-	"github.com/bitrise-tools/go-xamarin/constants"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"time"
 )
 
 // ConfigsModel ...
 type ConfigsModel struct {
-	XamarinSolution string
-	NugetVersion    string
+	XamarinSolution string `env:"xamarin_solution,file"`
+	NuGetVersion    string `env:"nuget_version"`
+	AllowCaching    string `env:"allow_caching,opt[local,global,none]"`
```

Comment:
use cache level: https://github.com/bitrise-steplib/bitrise-step-android-unit-test/blob/master/step.yml#L105

Reply:
Renamed to cache level.